_includes/footer.html

```diff
@@ -42,6 +42,16 @@
                         </a>
                     </li>
                     {% endif %}
+                    {% if site.linkedin_username %}
+                    <li>
+                        <a href="https://linkedin.com/in{{ site.linkedin_username }}">
```

Comment:
There is a typo here.  The url needs a / after in:

<a href="https://linkedin.com/in/{{ site.linkedin_username }}">